money-monkey-web/app/ui/PopupModal.tsx

```
@@ -0,0 +1,57 @@
+        visible ? "bg-gray-200 bg-opacity-50" : "pointer-events-none"
+      }`}
+      onClick={onClose}
+    >
+      <div
+        className={`transition-all flex justify-center flex-col gap-1 pt-2 p-4 bg-white border-2 rounded-xl [&_label]:pr-2 ${
+          visible ? "scale-100" : "scale-0"
+        }`}
+        onClick={(e) => {
+          e.stopPropagation();
+        }}
+      >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)